README.md

```diff
@@ -1089,15 +1089,32 @@ Resource tags specified with `--tags` will be added to new IAM Roles and new or
 The ECS CLI supports creating, running, inspecting and stopping tasks defined by an ECS Task Definition through its `local` subcommands. You can run an ECS Task Definition specified in a local JSON file or pulled from a registered ECS Task Definition.
 
 #### ecs-cli local create
+If you want to convert an ECS Task Definition to a Docker Compose file, you can run:
+
+```
+$ ecs-cli local create
+```
+Without arguments, this will try to read an ECS Task Definition from local a file named `task-definition.json` located in the current directory and generate both a compose file, by default named `docker-compose.ecs-local.yml`, as well as a compose override file, by default named `docker-compose.ecs-local.override.yml`. You can then call `ecs-local up`.
+**NOTE** Using these Compose files as input to `ecs-cli compose` subcommands may not translate back to the same ECS Task Definition used as input to `local create`.
+
+If you wish to run an ECS Task Definition specified in a different file, you can use the `--task-def-file` or `-f` flag with the name of the file.
+If you wish to run an ECS Task Definition already registered with ECS, you can use the `--task-def-remote` or `-t` flag with the ARN or family name of the Task Definition.
+You can also specify a different output file using the `--output` or `-o` flag.
+
 
 #### ecs-cli local up
+If you want to convert an ECS Task Definition to a Docker Compose file, you can run:
+
+```
+$ ecs-cli local create
```

Comment:
1. Should this be `$ ecs-cli local up`?
2. Should we put more info about `up`, since `up` not only "convert" but also "run" containers?

Reply:
Agreed, I think we can merge this PR for "local create" and create a separate one for "local up". What do you think @SoManyHs ?

For local up, it'd be nice to tell users about:
- The [Amazon ECS Local Container Endpoints](https://github.com/awslabs/amazon-ecs-local-container-endpoints) and the `ecs-local-network`.
- Overrides as a way of replacing a container that's in a task definition with a local one.

Reply:
> Should this be $ ecs-cli local up?

I mean, this does all apply for create. I do think that we can flesh out local up more, but if the implication here is that the whole description should only apply to up I'm not sure I agree.

Reply:
Oh. I just realized my PR is not what I have latest locally. Plz hold